### PR TITLE
feat(LayoutMenu): Creates LayoutMenu using Flyout and Accordions

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Creates new `LayoutMenu` component that utilizes `Flyout`, `Accordion`, and `Button` to create a menu that is accordions on desktop and a button into flyout with accordions on mobile/tablet. This is a generalized version of the catalog menu.

Check it out at https://60bfb9c6b597b623ed9a2932--gamut-preview.netlify.app/storybook/?path=/docs/organisms-layoutmenu--layout-menu

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [REACH-837](https://codecademy.atlassian.net/browse/REACH-837)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.20f0fa.0<br/>@codecademy/gamut-labs@18.2.2-alpha.20f0fa.0<br/>@codecademy/styleguide@40.0.7-alpha.20f0fa.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae75184f409022433ac847--gamut-preview.netlify.app>https://60ae75184f409022433ac847--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae75184f409022433ac847>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.0971a9.0<br/>@codecademy/gamut-labs@18.2.2-alpha.0971a9.0<br/>@codecademy/styleguide@40.0.7-alpha.0971a9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae97b43e288800c62987d5--gamut-preview.netlify.app>https://60ae97b43e288800c62987d5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae97b43e288800c62987d5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.048a54.0<br/>@codecademy/gamut-labs@18.2.2-alpha.048a54.0<br/>@codecademy/styleguide@40.0.7-alpha.048a54.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ae9c220d0c8207b3294d00--gamut-preview.netlify.app>https://60ae9c220d0c8207b3294d00--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ae9c220d0c8207b3294d00>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.c1b04d.0<br/>@codecademy/gamut-labs@18.2.2-alpha.c1b04d.0<br/>@codecademy/styleguide@40.0.7-alpha.c1b04d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60aeb342dc093b00c00bd2a0--gamut-preview.netlify.app>https://60aeb342dc093b00c00bd2a0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60aeb342dc093b00c00bd2a0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.6fedce.0<br/>@codecademy/gamut-labs@18.2.2-alpha.6fedce.0<br/>@codecademy/styleguide@40.0.7-alpha.6fedce.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60aebbc1857bbf00c4b34a68--gamut-preview.netlify.app>https://60aebbc1857bbf00c4b34a68--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60aebbc1857bbf00c4b34a68>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.b08503.0<br/>@codecademy/gamut-labs@18.2.2-alpha.b08503.0<br/>@codecademy/styleguide@40.0.7-alpha.b08503.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60aedc4ba966fb00d8c9936a--gamut-preview.netlify.app>https://60aedc4ba966fb00d8c9936a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60aedc4ba966fb00d8c9936a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.6c552e.0<br/>@codecademy/gamut-labs@18.2.2-alpha.6c552e.0<br/>@codecademy/styleguide@40.0.7-alpha.6c552e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60af12d6a1d23963012c561e--gamut-preview.netlify.app>https://60af12d6a1d23963012c561e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60af12d6a1d23963012c561e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.accb4d.0<br/>@codecademy/gamut-labs@18.2.2-alpha.accb4d.0<br/>@codecademy/styleguide@40.0.7-alpha.accb4d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60aff39bb8436d132c43a06f--gamut-preview.netlify.app>https://60aff39bb8436d132c43a06f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60aff39bb8436d132c43a06f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.d349b3.0<br/>@codecademy/gamut-labs@18.2.2-alpha.d349b3.0<br/>@codecademy/styleguide@40.0.7-alpha.d349b3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b00860c563c823531847fa--gamut-preview.netlify.app>https://60b00860c563c823531847fa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b00860c563c823531847fa>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.8a6f31.0<br/>@codecademy/gamut-labs@18.2.2-alpha.8a6f31.0<br/>@codecademy/styleguide@40.0.7-alpha.8a6f31.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b025fbb8436d554a43a269--gamut-preview.netlify.app>https://60b025fbb8436d554a43a269--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b025fbb8436d554a43a269>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.43ce54.0<br/>@codecademy/gamut-labs@18.2.2-alpha.43ce54.0<br/>@codecademy/styleguide@40.0.7-alpha.43ce54.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b133bbf36a24742c5146b3--gamut-preview.netlify.app>https://60b133bbf36a24742c5146b3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b133bbf36a24742c5146b3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.6e942b.0<br/>@codecademy/gamut-labs@18.2.2-alpha.6e942b.0<br/>@codecademy/styleguide@40.0.7-alpha.6e942b.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.fecc9b.0<br/>@codecademy/gamut-labs@18.2.2-alpha.fecc9b.0<br/>@codecademy/styleguide@40.0.7-alpha.fecc9b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b161baa8348522e6d84ba9--gamut-preview.netlify.app>https://60b161baa8348522e6d84ba9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b161baa8348522e6d84ba9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.119-alpha.50f33e.0<br/>@codecademy/gamut-labs@18.2.2-alpha.50f33e.0<br/>@codecademy/styleguide@40.0.7-alpha.50f33e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60b1634a5fa6c426e62538e0--gamut-preview.netlify.app>https://60b1634a5fa6c426e62538e0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60b1634a5fa6c426e62538e0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.7-alpha.b481a9.0<br/>@codecademy/gamut-labs@18.4.1-alpha.b481a9.0<br/>@codecademy/styleguide@40.3.5-alpha.b481a9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ba354e00c531361721597f--gamut-preview.netlify.app>https://60ba354e00c531361721597f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ba354e00c531361721597f>Deploy Logs</a></p>

File: packages/gamut-labs/src/LayoutMenu/AccordionMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        return (
+          <Box key={item.slug} py={8}>
+            <SectionItemLink
+              onClick={() => {
+                onSectionItemClick(item, section.slug);
+              }}
+            >
+              {item.title}
+            </SectionItemLink>
+          </Box>
```
[the remark this answers is not in the document]
Comment (aresnik11) [reply packages/gamut-labs/src/LayoutMenu/AccordionMenu.tsx:75]: i believe this is the part that will become your generalized menu base component? so going to leave as is until that is ready

[the remark this answers is not in the document]
Comment (codecaaron) [reply packages/gamut-labs/src/LayoutMenu/AccordionMenu.tsx:75]: Yep!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.13-alpha.bbee7a.0<br/>@codecademy/gamut-labs@19.0.6-alpha.bbee7a.0<br/>@codecademy/styleguide@41.2.4-alpha.bbee7a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60bfb9c6b597b623ed9a2932--gamut-preview.netlify.app>https://60bfb9c6b597b623ed9a2932--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60bfb9c6b597b623ed9a2932>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.14-alpha.53513c.0<br/>@codecademy/gamut-labs@19.1.1-alpha.53513c.0<br/>@codecademy/styleguide@41.3.1-alpha.53513c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c13bee4876f11042cee10d--gamut-preview.netlify.app>https://60c13bee4876f11042cee10d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c13bee4876f11042cee10d>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
🚢

**Review by christian-dinh — APPROVED** [review]
Looking at how this is consumed in https://github.com/codecademy-engineering/portal-app/pull/82 it works really nicely! I'm curious how we would handle a layout like the cheatsheet homepage, in which the flyout menu button is not at the top of the right column content. It's something that could maybe be done by changing the flex order or maybe would require splitting this into two separate components (`Sidebar` and `Flyout` versions), but either way that's something we can address when it becomes relevant

Screen Shot 2021-06-10 at 12 06 46 PM

**Comment by aresnik11** [comment]
@christian-dinh we got the ok for the catalog to move the flyout menu button to be the first item (it previously was like cheatsheets) so that would also hopefully be the case for other pages
>I'm curious how we would handle a layout like the cheatsheet homepage, in which the flyout menu button is not at the top of the right column content. It's something that could maybe be done by changing the flex order or maybe would require splitting this into two separate components (Sidebar and Flyout versions), but either way that's something we can address when it becomes relevant

**Comment by christian-dinh** [comment]
oh nice well nvm then 🎉

File: packages/gamut-labs/src/LayoutMenu/AccordionMenu.tsx (hunk 1 above)
Comment (MattSchiller) [inline packages/gamut-labs/src/LayoutMenu/AccordionMenu.tsx:75]: Minor nit, could we nest the gate logic a layer down so we don't have the duplicated wrapper?
```suggestion
      {section.items.map((item) => {
          return (
            <Box key={item.slug} py={8}>
              { (selectedItem === item.slug)
               ? <SelectedSectionItem>{item.title}</SelectedSectionItem>
               : <SectionItemLink onClick={() => onSectionItemClick(item, section.slug) }>
                  {item.title}
                 </SectionItemLink>
               }
          </Box>
```

**Review by MattSchiller — APPROVED** [review]
Looks good, minor nit on a render function.

I think there's still an open question in the portal-app PR if we want to include NextJS Link logic in here at all, which would be my only hesitation on merging this.

File: packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  sections,
+  onSectionToggle,
+  onSectionItemClick,
+  selectedItem,
+  mobileButtonText,
+  fullWidthButton = true,
+  showLogoOnFlyout,
+  children,
+}) => {
+  const [flyoutOpen, toggleFlyout] = useState<boolean>(false);
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx:48]: Nit: TypeScript can infer this generic type from the `false`

```suggestion
  const [flyoutOpen, toggleFlyout] = useState(false);
```

File: packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+   */
+  mobileButtonText: string;
+  /**
+   * Whether the mobile button should be full width
+   */
+  fullWidthButton?: boolean;
+  /**
+   * Whether or not to show the Codecademy logo at the top of the flyout on mobile
+   */
+  showLogoOnFlyout?: boolean;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx:35]: Are we sure we're not always going to have this? Or failing that, are we sure it's always going to be either the logo or nothing? It'd be nice if we could remove this prop and assume standardization...

Same with `fullWidthButton`?

File: packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+};
+
+export const LayoutMenu: React.FC<LayoutMenuProps> = ({
+  sections,
+  onSectionToggle,
+  onSectionItemClick,
+  selectedItem,
+  mobileButtonText,
+  fullWidthButton = true,
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx:44]: Nit: I've always found it confusing (an anecdotally know others have as well) to see props that default to truthy if not provided. Can this be changed to something like collapsed/narrow?

File: packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            {showLogoOnFlyout && <Logo mb={32} />}
+            {sections.map((section) => (
+              <AccordionMenu
+                key={section.slug}
+                section={section}
+                onSectionToggle={onSectionToggle}
+                onSectionItemClick={(
+                  item: SectionItem,
+                  sectionSlug: string
+                ) => {
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx:75]: Nit: TypeScript should be able to infer these, no?

```suggestion
                onSectionItemClick={(item, sectionSlug) => {
```

File: packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,119 @@
+import { Box } from '@codecademy/gamut';
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: Is there work slated anywhere to spruce up the `Flyout`'s close button? It looks a little overlappy-y here:

Screenshot of an X button overlapping white and light gray

**Review by JoshuaKGoldberg — APPROVED** [review]
Looks great! Some touchup requests here or there, but nothing major 😎

File: packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx (hunk 3 above)
Comment (aresnik11) [reply packages/gamut-labs/src/LayoutMenu/LayoutMenu.tsx:35]: was trying to make this as configurable as possible but really for reach team's use cases we always want the logo and the button to be full width

File: packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx (hunk 6 above)
Comment (aresnik11) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: oooh this did not look like this before, thanks for catching! @rogervera can you address this in the flyout PR https://github.com/Codecademy/client-modules/pull/1740

Comment (rogervera) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: Will do!

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c2784ac097d3008f610cb1--gamut-preview.netlify.app>https://60c2784ac097d3008f610cb1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c2784ac097d3008f610cb1>Deploy Logs</a></p>

File: packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx (hunk 6 above)
Comment (rogervera) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: Josh, Amy what were the steps to reproduce? I can't reproduce using the Chrome responsive tool, Storybook canvas tool, or on mobile (iPhone XR - 828px).

Was this on a specific viewport width range?

Comment (JoshuaKGoldberg) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: Aha I'm not using any kind of responsive tool, just resizing my browser window.

Screenshot of a Brave browser at medium width with the overlap bug

Comment (rogervera) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: Thanks Josh. I found out what my problem was on my end. :face_palm:
I had my screen resolution set to a larger size where I couldn't reproduce.

Thank you!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.9578c9.0<br/>@codecademy/gamut-labs@19.5.1-alpha.9578c9.0<br/>@codecademy/styleguide@41.7.5-alpha.9578c9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3a408f98c23098f30c35a--gamut-preview.netlify.app>https://60c3a408f98c23098f30c35a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3a408f98c23098f30c35a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.321d22.0<br/>@codecademy/gamut-labs@19.5.1-alpha.321d22.0<br/>@codecademy/styleguide@41.7.5-alpha.321d22.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3b0423464ea144c55c407--gamut-preview.netlify.app>https://60c3b0423464ea144c55c407--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3b0423464ea144c55c407>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.271e04.0<br/>@codecademy/gamut-labs@19.5.1-alpha.271e04.0<br/>@codecademy/styleguide@41.7.5-alpha.271e04.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3b51269ec5c0090d91424--gamut-preview.netlify.app>https://60c3b51269ec5c0090d91424--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3b51269ec5c0090d91424>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.f6acfb.0<br/>@codecademy/gamut-labs@19.5.1-alpha.f6acfb.0<br/>@codecademy/styleguide@41.7.5-alpha.f6acfb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3c2d91230920a2faf86f3--gamut-preview.netlify.app>https://60c3c2d91230920a2faf86f3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3c2d91230920a2faf86f3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.1b651d.0<br/>@codecademy/gamut-labs@19.5.1-alpha.1b651d.0<br/>@codecademy/styleguide@41.7.5-alpha.1b651d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3c882c9cc93134c91dbc7--gamut-preview.netlify.app>https://60c3c882c9cc93134c91dbc7--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3c882c9cc93134c91dbc7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.f0a35a.0<br/>@codecademy/gamut-labs@19.5.1-alpha.f0a35a.0<br/>@codecademy/styleguide@41.7.6-alpha.f0a35a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c3da81d0d30d24ca9c1895--gamut-preview.netlify.app>https://60c3da81d0d30d24ca9c1895--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c3da81d0d30d24ca9c1895>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
lgtm, awesome work!!

**Review by christian-dinh — APPROVED** [review]
re-shipping, code looks great to me 👍 

did we ever figure out how we're gonna get next routing to work with this? I vaguely remember some slack messages but I don't remember where

**Comment by aresnik11** [comment]
> re-shipping, code looks great to me 👍
> 
> did we ever figure out how we're gonna get next routing to work with this? I vaguely remember some slack messages but I don't remember where

@christian-dinh that was handled in this commit https://github.com/Codecademy/client-modules/pull/1784/commits/efec32542c888dd9fd20c298612f666502038803!

File: packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx (hunk 6 above)
Comment (rogervera) [reply packages/styleguide/stories/Organisms/LayoutMenu.stories.mdx:1]: https://codecademy.atlassian.net/browse/REACH-1055 Ticket for this

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.23-alpha.85a352.0<br/>@codecademy/gamut-labs@19.5.1-alpha.85a352.0<br/>@codecademy/styleguide@41.7.6-alpha.85a352.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c7b9e997efeb0aa82a0450--gamut-preview.netlify.app>https://60c7b9e997efeb0aa82a0450--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c7b9e997efeb0aa82a0450>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]
🎉

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.27-alpha.b953d2.0<br/>@codecademy/gamut-labs@19.6.2-alpha.b953d2.0<br/>@codecademy/styleguide@41.7.12-alpha.b953d2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c9053772003b0f2b7081f5--gamut-preview.netlify.app>https://60c9053772003b0f2b7081f5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c9053772003b0f2b7081f5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.27-alpha.74d090.0<br/>@codecademy/gamut-labs@19.6.2-alpha.74d090.0<br/>@codecademy/styleguide@41.7.12-alpha.74d090.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.27-alpha.95ea47.0<br/>@codecademy/gamut-labs@19.6.2-alpha.95ea47.0<br/>@codecademy/styleguide@41.7.12-alpha.95ea47.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c91083701fda21f56056a5--gamut-preview.netlify.app>https://60c91083701fda21f56056a5--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c91083701fda21f56056a5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.29-alpha.37bd07.0<br/>@codecademy/gamut-labs@19.7.1-alpha.37bd07.0<br/>@codecademy/styleguide@41.8.1-alpha.37bd07.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60c919444e42f02b6601811c--gamut-preview.netlify.app>https://60c919444e42f02b6601811c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60c919444e42f02b6601811c>Deploy Logs</a></p>

**Pull request merged**